it/uk/gov/hmrc/soletraderidentificationfrontend/controllers/BusinessVerificationControllerISpec.scala

```diff
@@ -71,14 +71,17 @@ class BusinessVerificationControllerISpec extends ComponentSpecHelper
         stubRetrieveBusinessVerificationResultFromStub(testBusinessVerificationJourneyId)(OK, Json.obj("verificationStatus" -> "PASS"))
         stubStoreBusinessVerificationStatus(journeyId = testJourneyId, businessVerificationStatus = BusinessVerificationPass)(status = OK)
         stubRetrieveBusinessVerificationStatus(testJourneyId)(OK, Json.toJson(BusinessVerificationPass))
+        stubRetrieveSoleTraderDetails(testJourneyId)(OK, Json.toJson(testSoleTraderDetails(true)))
         stubRetrieveNino(testJourneyId)(OK, testNino)
+        stubRetrieveSAPostcode(testJourneyId)(NOT_FOUND)
         stubRetrieveSautr(testJourneyId)(OK, testSautr)
         stubRegister(testNino, testSautr)(OK, Registered(testSafeId))
         stubStoreRegistrationStatus(testJourneyId, Registered(testSafeId))(OK)
         stubAudit()
```

Comment:
All these calls after the stubAudit() are for the audit service. With the new refactor we could remove a lot of these to make it a bit smaller. Could also be removed from the other IT specs that stub the audit as well

Reply:
Fixed